src/components/search/search.tsx

```diff
@@ -65,11 +65,20 @@ const Search = () => {
             placeholder: "Search keyword(s)",
           }}
           sx={{
-            [`@media screen and (min-width: ${headerBreakpoints.lgMobile})`]: {
-              flexFlow: "column nowrap",
-            },
-            [`@media screen and (min-width: ${headerBreakpoints.smTablet})`]: {
-              flexFlow: "row nowrap",
+            flexFlow: "row nowrap",
```

Comment:
The below code is weird but I think necessary to get around the component styling, particularly the existence of the "Search" text

Reply:
Yeah, the `SearchBar` doesn't give you much room for this type of custom styling. It seems reasonable but why are textTransform and borderRadius needed? Two two don't do much, especially textTransform since the text is moved outside of the visible screen and is already capitalized.

The only update I'd ask is to update this to use `min-width` instead of `max-width` to be more consistent with the mobile-first approach.

Reply:
`textTransform` was actually not needed– holdover from a different approach. `borderRadius` is necessary, needs to be explicitly set I think because the `textIndent` overrides the borders of the button.  I know we are aiming for a mobile-first approach but this is a situation where it's really counterintuitive, since the styling is meant to undo what's already there. Using `min-width` here adds 10-15 unnecessary lines of code, basically just trying to restore the search bar to its own default styling @EdwinGuzman

Reply:
Great, thanks for the explanation and trying it out!